cal/schema.py

```diff
@@ -42,3 +42,6 @@ class User(db.Model):
     name = db.Column(db.String, unique=True, nullable=False)
 
     events = db.relationship("Event", backref="user")
+
+    def to_json(self):
```

Comment:
if this is meant to return JSON it should be returning a string, not a dict.
(same for the `Event` class)